.github/workflows/unit_test_workflow.yml

```diff
@@ -96,3 +98,17 @@ jobs:
         run: |
           cd OpenSearch-Dashboards/plugins/dashboards-assistant
           yarn run test:jest --coverage
+
+  tests-codecov:
+    needs: tests-linux
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: coverage-report-linux
+          path: ./
+      - name: Upload Coverage Report
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          files: ./clover.xml
```

Comment:
should this files match with line 56 ` path: ./OpenSearch-Dashboards/plugins/dashboards-assistant/coverage/clover.xm`

Reply:
No need @Hailong-am since only that one clover.xml is zipped without folder structure.
You can see the artifacts here:
https://github.com/opensearch-project/dashboards-assistant/actions/runs/14385335836